apps/docs/src/app/platform/component-docs/platform-input/platform-input-docs.component.html

```diff
@@ -0,0 +1,17 @@
+<fd-docs-section-title [id]="'inputdef'" [componentName]="'input'">
+	Input
+</fd-docs-section-title>
+<description>This example shows Input default implementaion </description>
+<component-example>
+	<fd-platform-input-example></fd-platform-input-example>
+</component-example>
+<code-example [exampleFiles]="defaultInputType"></code-example>
+
+<fd-docs-section-title [id]="'inputValid'" [componentName]="'inputValidation'">
+	Input States with Messages Components.
```

Comment:
Should be 'Input States With Message Components' (all words should be capitalized without punctuation) :)